purge-job/base/kustomization.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  app.kubernetes.io/name: thoth
+  app.kubernetes.io/component: purge-job
+  app.kubernetes.io/managed-by: aicoe-thoth-devops
+generatorOptions:
+  disableNameSuffixHash: true
+resources:
+  - imagestream.yaml
+  - argo-workflows/purge-template.yaml
+  - openshift-templates/purge-template.yaml
```

oh the name different.
```suggestion
  - openshift-templates/purge.yaml
```

---

Thanks, adjusted 👍🏻